### ISSUE TITLE
Support transactions on SORA network

### ISSUE THREAD

**Issue by smolicz** [Zondax/ledger-polkadot#85]
Currently polkadot application does not support SORA network. We can on other sends the fund there but once they are there it is blocked. Would it be possible to whitelist this network to be supported by Ledger polkadot application? https://polkaswap.io/#/bridge

**Comment by jleni** [comment]
Ledger applications for Substrate chains are not generic. Each chain has its own structure and metadata.
To support another network like Sora, we oudl need to create a complete new app.

This is not within the scope of this repo.

**Comment by smolicz** [comment]
Thanks for the info. That means my funds are basically lost, right?

**Comment by Integritus** [comment]
@jleni How different are they? Would it be possible to say, fork this project and refactor for Sora implementation, or would it be a from scratch effort?

**Comment by smolicz** [comment]
Found solution how to get my funds back. Extracted polkadot seed from ledger and import to the metamask.

**Comment by Integritus** [comment]
@smolicz How did you do that? I tried something along those lines and didn't had success.

**Comment by smolicz** [comment]
https://www.reddit.com/r/SORA/comments/n6436b/ledger_nano_x_problem_poldotjs_on_polkaswap/

Main thing is to export the seed via this project: https://github.com/jacogr/sample-ledger-ed25519

Once you have the seed import it via polkadot.js website (raw seed) and change the encryption for second option sdxxxx. The seed might be generated for the different address in your ledger(it was my case). Luckily I was able to find out that changing the last digit of the seed switches the account 🤣 Once this was done I exported the json and imported to my metamask. Bum! All money back now :)
